apps/aebytecode/src/aebytecode.app.src

```diff
@@ -1,15 +0,0 @@
-{application, aebytecode,
- [{description, "Bytecode definitions for AEthernity VM shared with compiler."},
```

Comment:
aeternity - though cosmetic.

Reply:
I just copied it ;-). Will make a new PR for that... not so urgent.